packages/webapp/pages/notifications.tsx

```diff
@@ -119,8 +119,7 @@ const Notifications = (): ReactElement => {
     }
 
     setNotification(item);
-    const { right, bottom } = e.currentTarget.getBoundingClientRect();
-    show(e, { position: { x: right, y: bottom + 4 } });
+    onMenuClick(e);
```

Comment:
Noticed u rename this in other files, and not here.. any reason? Let's be consistent maybe and rename also here based on the menu it opens..